lib/pack.js

```diff
@@ -163,6 +163,7 @@ function packGitDep (manifest, dir) {
       const child = cp.spawn(process.env.NODE || process.execPath, [
         require.main.filename,
         'install',
+        '--dev',
```

Comment:
I think there should be a `--prod` here as well to protect us against users who were doing an `--only=dev` install previously.